udata_hydra/routes/__init__.py

```diff
@@ -42,14 +48,11 @@ def generate_routes(
     (web.get, "/api/checks/all", get_all_checks, None),
     (web.post, "/api/checks", create_check, None),
     # Routes for resources
-    (web.get, "/api/resources", get_resource, None),
+    (web.get, "/api/resources/{resource_id}", get_resource, None),
     (web.get, "/api/resources/{resource_id}/status", get_resource_status, None),
     (web.post, "/api/resources", create_resource, None),
```

Comment:
I wonder if we should update the payload on this route at some point, but would probably be done in a future PR if needed.